### ISSUE TITLE
Run update-mime-database and update-desktop-database

### ISSUE THREAD

**Issue by probonopd** [AppImage/appimaged#34]
According to https://discourse.appimage.org/t/how-to-use-linuxdeployqt/275/15, `appimaged` should run 

```
update-mime-database $HOME/.local/share/mime
update-desktop-database $HOME/.local/share/applications
```

(if these commands are available on the `$PATH`) in order to work correctly.

__Note:__ This should _not_ be done after _each_ AppImage processed by _appimaged_ in order to save CPU power, but only after the system has quiesced (e.g., no further AppImages have been processed in the last X seconds).

**Comment by probonopd** [comment]
Let's use GitHub's shiny new "Transfer Issue" beta feature for this. :+1:

**Comment by TheAssassin** [comment]
That's what was used, wasn't it?

**Comment by probonopd** [comment]
No, at least it didn't seem to have worked for me... I now did it.
